elephant_vending_machine/views.py

```diff
@@ -38,27 +42,74 @@ def run_trial():
         response = 'No trial_name specified'
     return response
 
+def allowed_file(filename):
+    """Determines whether an uploaded image file has an allowed extension.
+
+    Returns:
+        True if filename includes extension and extension is an allowed extension
+        False otherwise
+    """
+    return '.' in filename and \
+           filename.rsplit('.', 1)[1].lower() in ALLOWED_EXTENSIONS
+
 
-@APP.route('/add-image', methods=['POST'])
+@APP.route('/image', methods=['POST'])
 def upload_image():
-    """Responds with 'File received' string
+    """Return string indicating result of request
+
+    **Example request**:
+
+    .. sourcecode:: http
```

Comment:
```suggestion
    .. sourcecode::
```

Sphinx can't parse this as HTML since it's not actually formatted as HTML. If you remove this, it'll look the same but won't throw the error on doc build